docs/source/torch.rst

```diff
@@ -183,217 +223,244 @@ Examples::
   >>> y.requires_grad
   False
 
-.. autofunction:: no_grad
-.. autofunction:: enable_grad
-.. autofunction:: set_grad_enabled
+.. autosummary::
+    :toctree: generated
+    :nosignatures:
+
+    no_grad
+    enable_grad
+    set_grad_enabled
 
 Math operations
 ----------------------------------
 
-Pointwise Ops
-~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
I think you might have removed the sub-title? 

Here are the screenshots of the sidebar for torch.html

| Before | After | 
| --- | --- |
| ![image](https://user-images.githubusercontent.com/6421097/80798842-f3205c80-8b6a-11ea-93df-5c59de61b300.png) | ![image](https://user-images.githubusercontent.com/6421097/80798928-206d0a80-8b6b-11ea-9bdc-a251ecf352b0.png) |

Reply:
fixed in cab782a